packages/kit-bg/src/vaults/impls/sui/sdkSui/transactions.ts

```
@@ -0,0 +1,267 @@
+  while (hasNextPage && retries < maxRetries) {
+    try {
+      const resp = await client.getCoins({
+        owner: address,
+        coinType,
+        cursor,
+        limit: 50,
+      });
+
+      const { data, nextCursor, hasNextPage: nextPageExists } = resp;
+
+      if (data && data.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)